sale_layout_category_hide_detail/models/account_invoice.py

```diff
@@ -13,3 +13,4 @@ class AccountInvoiceLine(models.Model):
     show_subtotal = fields.Boolean(
         string="Show subtotal",
         default=True)
+    show_price = fields.Boolean(string="Show Price", default=True)
```

Comment:
```suggestion
    show_price = fields.Boolean(
        string="Show Price",
        default=True)
```

To match the code style